docker-orchestration/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.9-slim-buster
+
+RUN apt-get update \
+    && apt-get -q -y install git gcc nginx python3-libvirt libxslt1-dev libxml2-dev qemu-kvm libvirt-dev virt-manager mediainfo
+
+WORKDIR /var/www
+RUN git clone https://github.com/CCExtractor/sample-platform.git
```

Comment:
I wouldn't clone the SP in the docker build, but have it passed to the docker instance through a volume. That'll make it easier for us to update (I think)